<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalcWindow</class>
 <widget class="QMainWindow" name="CreditCalcWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>549</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Rounded MT Bold</family>
    <pointsize>21</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Credit Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDoubleSpinBox" name="doubleSpinBox_credit_total_amount">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>200</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	border : 2px solid rgb(41, 50, 66);
	background : rgb(58, 70, 90);
}</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_credit_term">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>200</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
	border : 2px solid rgb(41, 50, 66);
	background : rgb(58, 70, 90);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_credit_total_amount">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>221</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Total credit amount</string>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_credit_term">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>70</y>
      <width>161</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Credit term</string>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_term_mode">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>140</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border: 1px solid rgb(31, 38, 50);
	background-color:rgb(114, 215, 151);
	color: white;
}</string>
    </property>
    <item>
     <property name="text">
      <string>months</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>years</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_credit_rate">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>120</y>
      <width>171</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Credit rate</string>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_annuity">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>175</y>
      <width>181</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Annuity</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_differentiated">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>171</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Differentiated</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_credit_payment_type">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>175</y>
      <width>200</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Type of monthly payments</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_calculate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>200</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(31, 38, 50);
	background-color:rgb(114, 215, 151);
	color: white;
}

QPushButton:pressed {
	background-color: rgb(104, 196, 138);
}</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_credit_rate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>200</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	border : 2px solid rgb(41, 50, 66);
	background : rgb(58, 70, 90);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_credit_monthly_payment">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>340</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Monthly payment</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_credit_overpayment">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>390</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Overpayment on credit</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_credit_total_payment">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>460</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Total payment</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_value_credit_total_payment">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>350</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_value_credit_overpayment">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>350</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_value_monthly_payment">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>350</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>11</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_currency_mode">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>140</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border: 1px solid rgb(31, 38, 50);
	background-color:rgb(114, 215, 151);
	color: white;
}</string>
    </property>
    <item>
     <property name="text">
      <string>RUB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>USD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EUR</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_percent">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
